sys/arm64/include/elf.h

```diff
@@ -93,6 +93,8 @@ __ElfType(Auxinfo);
 
 #define	ELF_MACHINE_OK(x) ((x) == (ELF_ARCH))
 
+#define	ELF_IS_CHERI(hdr) (((hdr)->e_entry & 1) == 1)
```

Comment:
Moving this change to #840 which is where this logically belongs (and building a Morello userspace is dependent on a Morello kernel IMO given you ned kernel headers).